package.json

```diff
@@ -89,6 +90,7 @@
     "proclaim": "3.4.4",
     "q": "1.4.1",
     "query-string": "4.2.3",
+    "querystring": "^0.2.0",
```

Comment:
Can you use query-string instead (since we already have it just above) to limit the number of dependencies. Thanks!